src/components/AdminDashboard/StatsCard.jsx

```diff
@@ -12,6 +12,7 @@ import {
   IconButton,
   useDisclosure,
   Button,
+  calc,
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'calc' is defined but never used.